azure-core/azure/core/pipeline/transport/requests_asyncio.py

```diff
@@ -70,6 +70,7 @@ class AsyncioRequestsTransport(RequestsTransport, AsyncHttpTransport):  # type:
     async def send(self, request: HttpRequest, **kwargs: Any) -> AsyncHttpResponse:  # type: ignore
         """Send the request using this HTTP sender.
         """
+        self.open()
```

Comment:
This makes the transport "reopen:able"/self-healing if called after the session is closed. Is this what one would expect? And do we change who owns the session in open()?

Reply:
How about ContextManager? 
When user calls ContextManage.exit, we want to close the session right?
Hence we need A way to close the session.
But I agree with you, allowing the use of close is a concern.

Reply:
Yes the current Transport implementation is "re-usable" after it has been closed.
We can definitely change that if we would prefer a Transport to be "single-use".